### PR TITLE
Tag ArbFloats v0.1.10 [https://github.com/JuliaArbTypes/ArbFloats.jl]

### PR DESCRIPTION
**Pull request by JeffreySarnoff**
restored REQUIRES following advice of @tkelman 
removed @vectorize macros
put Nemo into the REQUIRES file
minor typo fixups

### CODE REVIEW

File: ArbFloats/versions/0.1.10/requires [hunk 1]
```diff
@@ -0,0 +1,4 @@
+julia 0.5.0
+Nemo
+BinDeps
+Compat
```
Comment (tkelman) [inline ArbFloats/versions/0.1.10/requires]: It's best to declare a minimum version dependency that implements the features you need.

**Comment by JeffreySarnoff** [comment]
closing and reopening to force checks

**Comment by JeffreySarnoff** [comment]
@tkelman I get this error

> ERROR: LoadError: LoadError: The following tags for ArbFloats have not been pushed to upstream repository: 0.1.10.
> To fix this, navigate to the package directory and run `git push --tags`.
> 
When I try `git push --tags`  this happens
```bash
jas@mint ~/.julia/v0.5/ArbFloats $ git push --tags
The authenticity of host 'github.com (192.30.253.113)' can't be established.
RSA key fingerprint is SHA256:nThbg6kXUpJWGl7E1IGOCspRomTxdCARLviKw6E5SY8.
Are you sure you want to continue connecting (yes/no)? yes
Warning: Permanently added 'github.com,192.30.253.113' (RSA) to the list of known hosts.
Permission denied (publickey).
fatal: Could not read from remote repository.

Please make sure you have the correct access rights
and the repository exists.
```
but `git tag -l` reports that v0.1.10 is at the remote repository

```bash
jas@mint ~/.julia/v0.5/ArbFloats $ git tag -l
v0.0.1
v0.1.1
v0.1.10
v0.1.2
v0.1.3
v0.1.4
v0.1.5
v0.1.6
v0.1.7
v0.1.8
v0.1.9
```

_What should I do_

**Comment by tkelman** [comment]
`git tag -l` only shows your local working copy. You need to resolve why you're getting `Permission denied (publickey).` You're probably using the wrong ssh key? Try adding an https remote and pushing the tags to that?

**Comment by JeffreySarnoff** [comment]
added the tag on github manually

**Comment by JeffreySarnoff** [comment]
@tkelman worked it out -- thank you

**Comment by JeffreySarnoff** [comment]
closing in favor of using AttoBot [https://github.com/JuliaLang/METADATA.jl/pull/7233](https://github.com/JuliaLang/METADATA.jl/pull/7233)

**Pull request closed without merging**